lib/pltr/v2/actions/ui.js

```diff
@@ -213,6 +217,18 @@ export function setCardDialogClose() {
   return { type: SET_CARD_DIALOG_CLOSE }
 }
 
+export function setBookDialogOpen(bookId, bookNumber) {
```

Comment:
This interface is too generic.  I see you used it in two different ways previously.
That puts the onus on knowing how to use it on the component (and the writer of the component).
Please break it down into the following two functions instead:
 1. `openEditBookDialog(bookId)`, and
 2. `openNewBookDialog()`.
As discussed elsewhere, the book number isn't necessary and makes this confusing for the next person who wants to use this action.